user-registry/core/src/main/java/org/openbase/bco/registry/user/core/UserRegistryController.java

```diff
@@ -77,6 +83,26 @@ public UserRegistryController() throws InstantiationException, InterruptedExcept
     }
 
     @Override
+    public void activate() throws InterruptedException, CouldNotPerformException {
+        super.activate();
+
```

Comment:
still needed?

Reply:
Is still needed. Else the startup fails. See #68